src/Data/Formatter/DateTime.purs

```diff
@@ -100,8 +104,10 @@ printFormatterF cb = case _ of
   Hours24 a → "HH" <> cb a
   Hours12 a → "hh" <> cb a
   Meridiem a → "a" <> cb a
-  Minutes a → "mm" <> cb a
-  Seconds a → "ss" <> cb a
+  Minutes a → "m" <> cb a
```

Comment:
@garyb For me it is just a fix, but not a breaking change. Comparing to `moment.js` `ss` and `mm` do mean two digit values, not a single one. And `purescript-formatters` did not support this in a proper way before. With this pull request this issue would be fixed. 

It would be nice to to have this pull request before any `0.11` updates. Because it does not depend on any new feature of `0.11`. But using `0.11` does mean really a breaking change...